### DISCUSSION TITLE
TypeError: fit() got an unexpected keyword argument 'train_dataloaders'

### DISCUSSION THREAD

**Discussion by erving666** [Lightning-AI/pytorch-lightning#13230] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Here's my env:
  python 3.8.13
  pytorch                   1.11.0          py3.8_cuda11.3_cudnn8.2.0_0    pytorch
  pytorch-lightning         0.8.5                      py_0    conda-forge

I came across this issuse:
<img width="773" alt="image" src="https://user-images.githubusercontent.com/66522270/171995482-49a7a145-4555-4166-89ce-c6f01955389d.png">

I Have done a search for similar issue, but can't solve it. Please help.

**Comment by akihironitta** [comment]
@erving666 I believe it's because the version you're using is too old.
- 1.6.x: https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#fit
- 0.8.5: https://pytorch-lightning.readthedocs.io/en/0.8.5/trainer.html#trainer-class
